pkgs/development/python-modules/pdf2docx/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, fetchFromGitHub
+, buildPythonPackage
+, pymupdf
+, fire
+, fonttools
+, numpy
+, opencv4
+, tkinter
+, python-docx
+}:
+
+buildPythonPackage rec {
+  pname = "pdf2docx";
+  version = "0.5.6";
+  src = fetchFromGitHub {
+    owner = "dothinking";
+    repo = "pdf2docx";
+    rev = "d792b312c18870bad8c16f7cbac3e48dd4683abc";
+    sha256 = "mJo7++JF/NEPI4/g/+8+UpXAR/pPgmr2Mlaaj79kRNQ=";
+  };
+
+  postPatch = ''
+    substituteInPlace requirements.txt \
+      --replace 'opencv-python>=4.5' ""
+  '';
+
+  propagatedBuildInputs = [
+    tkinter
+    pymupdf
+    fire
+    fonttools
+    numpy
+    opencv4
+    python-docx
+  ];
+
+  meta = with lib; {
+    description = "Convert PDF to DOCX";
+    homepage = "https://github.com/dothinking/pdf2docx";
```

Comment:
Would you please add meta.changelog?
https://github.com/dothinking/pdf2docx/releases/tag/${src.rev}